src/utils/validateAvaxBurnedAmountEtna.ts

```diff
@@ -0,0 +1,58 @@
+import type { Context } from '../vms/context/model';
+import {
+  isAddPermissionlessDelegatorTx,
+  isAddPermissionlessValidatorTx,
+  isAddSubnetValidatorTx,
+  isCreateChainTx,
+  isCreateSubnetTx,
+  isPvmBaseTx,
+  isExportTx as isPvmExportTx,
+  isImportTx as isPvmImportTx,
+  isRemoveSubnetValidatorTx,
+  isTransferSubnetOwnershipTx,
+} from '../serializable/pvm';
+import type { UnsignedTx } from '../vms/common';
+import type { FeeState } from '../vms/pvm';
+import { calculateFee } from '../vms/pvm/txs/fee/calculator';
+
+export const validateAvaxBurnedAmountEtna = ({
+  unsignedTx,
+  context,
+  burnedAmount,
+  feeState,
+}: {
+  unsignedTx: UnsignedTx;
+  context: Context;
+  burnedAmount: bigint;
+  feeState: FeeState;
+}): { isValid: boolean; txFee: bigint } => {
+  const tx = unsignedTx.getTx();
+
+  const expectedFee = calculateFee(
+    unsignedTx.getTx(),
+    context.platformFeeConfig.weights,
+    feeState.price < context.platformFeeConfig.minPrice
+      ? context.platformFeeConfig.minPrice
+      : feeState.price,
+  );
+
+  if (
```

Comment:
What should we do if someone submits an x-chain tx post etna? How will it be handled?

Reply:
<img width="841" alt="Screenshot 2024-10-09 at 3 44 49 PM" src="https://github.com/user-attachments/assets/6a1024bb-cb97-4bc6-b479-b0b4365f67bc">

<img width="570" alt="Screenshot 2024-10-09 at 3 44 59 PM" src="https://github.com/user-attachments/assets/e32cfc92-0f12-4bae-afc1-abb8ff797363">

this will use the old logic if  someone submits an x-chain post-etna without x-chain dynamic fee implemented.